artemis-server/src/main/java/org/apache/activemq/artemis/core/persistence/config/PersistedSecuritySetting.java

```diff
@@ -252,8 +252,12 @@ public void decode(final ActiveMQBuffer buffer) {
       browseRoles = buffer.readNullableSimpleString();
       createAddressRoles = buffer.readNullableSimpleString();
       deleteAddressRoles = buffer.readNullableSimpleString();
-      viewRoles = buffer.readNullableSimpleString();
```

Comment:
darn, I read the method readNullableSimpleString as doing the check for an non existent value, my bad,